hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/WithObjectID.java

```diff
@@ -61,10 +61,6 @@ public long getUpdateID() {
    * @param obId - long
    */
   public void setObjectID(long obId) {
-    if (this.objectID != 0) {
```

Comment:
Or adjust the condition to make `obId` equals `OBJECT_ID_DEFAULT` the only exception:

```
if (this.objectID != 0 && obId != OBJECT_ID_DEFAULT) {
```